app/templates/components/billing/select-plan.hbs

```diff
@@ -1,79 +1,97 @@
+                  <span>
+                    <SvgImage @name="icon-help" @class="icon-help" />
+                    <EmberTooltip @text="Free Plan is a monthly-based plan.&#10;The Credits will be replenished&#10;automatically every month." />
+                  </span>
+                {{/if}}
+              </p>
+              <p class='billing-plans__box-v2--price' data-test-selected-plan-price="true">
+                {{#if plan.isFree}}
+                  Free
+                {{else}}
+                  <span class='color-asphalt-grey'>
+                    Starting at
+                  </span>
+                  {{format-currency plan.startingPrice floor="true"}}
+                {{/if}}
+              </p>
+              <p class='billing-plans__box-v2--desc' data-test-selected-plan-credits="true">
+                <SvgImage @name="icon-passed" @class="icon blue" /> {{plan.privateCredits}} Credits
+              </p>
+              <p class='billing-plans__box-v2--desc' data-test-selected-plan-oss-credits="true">
+                <SvgImage @name="icon-passed" @class="icon blue" /> {{plan.publicCredits}} OSS Credits<span class='color-cement-grey'>/month</span>
+              </p>
+              <p class='billing-plans__box-v2--desc' data-test-selected-plan-users="true">
+                <SvgImage @name="icon-passed" @class="icon blue" />
+                {{#if plan.isUnlimitedUsers}}
+                  Unlimited unique users
+                {{else}}
+                  Up to {{plan.startingUsers}} unique users
+                {{/if}}
+              </p>
+              <p class='billing-plans__box-v2--desc' data-test-selected-plan-repos="true">
+                <SvgImage @name="icon-passed" @class="icon blue" /> Private &amp; Open-Source repos
               </p>
-              <p class='billing-plans__box--jobs' data-test-selected-plan-jobs="true">
-                {{pluralize plan.builds 'concurrent job'}}
+              <p class='billing-plans__box-v2--desc' data-test-selected-plan-os="true">
+                <SvgImage @name="icon-passed" @class="icon blue" /> Linux, Windows, macOS
               </p>
+              {{#if (eq plan.id this.subscription.plan.id)}}
+                <h4 class='billing-plans' data-test-current-plan-text>Your current plan</h4>
+              {{else}}
+                {{#if (not (eq plan.id this.selectedPlan.id))}}
+                  <button data-test-subscribe-button
+                    onclick={{action (mut this.selectedPlan) plan}}
+                    class='button--white billing-plans__box-v2--button'
+                  >
+                    Select plan
+                  </button>
+                {{else}}
+                  <button data-test-subscribe-button
+                    onclick={{action form.submit}}
+                    class='button--blue billing-plans__box-v2--button'
+                  >
+                    Select plan
+                  </button>
+                {{/if}}
```

Comment:
Now, when clicking on the button, it selects the plan on the first click and then advances to the ordering process on the second click to the same button. This two-clicks behavior feels excessive and unnecessary to me. Would it be better to both select the plan and advance to ordering in a single click? Otherwise we can change the button title for selected plan - something like "Proceed to ordering". In this case it will feel more like a confirmation, rather than something didn't work. Although, my vote is still for the single click approach, as that's what I would expect by clicking on the button. What do you think?

Reply:
I'm not entirely sure how to do these two actions at once in this case.
Would this be acceptable?
```JS
<button data-test-subscribe-button
  {{action (mut this.selectedPlan) plan on='mouseDown'}}
  onclick={{action form.submit}}
  class='button--white-and-teal billing-plans__box-v2--button'
>
  Select plan
</button>
```

Reply:
I think the most straight forward way would be to define an action on component:

```js
export default Component.extend({
  ...
  actions: {
    selectAndSubmit(plan, form) {
      this.set('selectedPlan', plan');
      form.submit();
    }
  }
  ...
})
```
And then bind it on click in template as usual.

You can also add a short delay with `later(() => form.submit(), delay)` for a visual effect if you'd like :)

How does it sound to you?